packages/stream-text/test/index.test.ts

```diff
@@ -82,4 +84,199 @@ describe('@xsai/stream-text', () => {
+          },
+        ],
+        seed: 42,
+        toolChoice: 'required',
+        tools: [weather],
+      })
+
+      const textResult = []
+      for await (const chunk of chunkStream) {
+        textResult.push(chunk)
+      }
+
+      expect(textResult.join('').length).greaterThan(0)
+    }, 20000)
+
+    it('stream step', async () => {
+      const { stepStream } = streamText({
+        ...ollama.chat('mistral-nemo'),
+        maxSteps: 2,
+        messages: [
+          {
+            content: 'You are a helpful assistant.',
+            role: 'system',
+          },
+          {
+            content: 'What is the weather in San Francisco? do not answer anything else.',
+            role: 'user',
+          },
+        ],
+        seed: 42,
+        toolChoice: 'required',
+        tools: [weather],
+      })
+
+      const stepResult = []
+      for await (const step of stepStream) {
+        stepResult.push(step)
+      }
+
+      expect(stepResult).toHaveLength(2)
+    }, 20000)
+
+    it('stream text', async () => {
+      const { textStream } = streamText({
+        ...ollama.chat('mistral-nemo'),
+        maxSteps: 2,
+        messages: [
+          {
+            content: 'You are a helpful assistant.',
+            role: 'system',
+          },
+          {
+            content: 'What is the weather in San Francisco? do not answer anything else.',
+            role: 'user',
+          },
+        ],
+        seed: 42,
+        toolChoice: 'required',
+        tools: [weather],
+      })
+
+      const textResult = []
+      for await (const text of textStream) {
+        textResult.push(text)
+      }
+
+      expect(textResult.join('').length).greaterThan(0)
+    }, 20000)
+
+    it('onChunk', async () => {
+      const chunks = []
+
+      const { textStream } = streamText({
+        ...ollama.chat('mistral-nemo'),
+        maxSteps: 2,
+        messages: [
+          {
+            content: 'You are a helpful assistant.',
+            role: 'system',
+          },
+          {
+            content: 'What is the weather in San Francisco? do not answer anything else.',
+            role: 'user',
+          },
+        ],
+        onChunk: (chunk) => {
+          chunks.push(chunk)
+        },
+        seed: 42,
+        toolChoice: 'required',
+        tools: [weather],
+      })
+
+      for await (const _ of textStream) {
+
+      }
+
+      expect(chunks.length).greaterThan(0)
+    }, 20000)
+
+    it('onStepFinish', async () => {
+      const steps = []
+
+      const { textStream } = streamText({
+        ...ollama.chat('mistral-nemo'),
+        maxSteps: 2,
+        messages: [
+          {
+            content: 'You are a helpful assistant.',
+            role: 'system',
+          },
+          {
+            content: 'What is the weather in San Francisco? do not answer anything else.',
+            role: 'user',
+          },
+        ],
+        onStepFinish: (step) => {
+          steps.push(step)
+        },
```

Comment:
```suggestion
        onStepFinish: (step) => steps.push(step),
```

Reply:
this will make eslint complain about returnType

Reply:
> this will make eslint complain about returnType

Maybe we can change the return type to `unknown`